lib/senkyoshi/models/outcome_definition.rb

```diff
@@ -10,7 +10,10 @@ def self.from(xml, category)
       outcome_definition.iterate_xml(xml)
     end
 
-    def initialize(category)
+    def initialize(category, id = nil, content_id = nil, asidataid = nil)
+      @id = id || Senkyoshi.create_random_hex
```

Comment:
Implement dummy id's on the specs instead of creating one here.  Real data won't be missing an id